### ISSUE TITLE
[Confirmed] 11.17.0-50 working in EUR

### ISSUE THREAD

**Issue by iCRON862** [zoogie/super-skaterhax#3]

**Comment by zoogie** [comment]
Thank you for the confirmation.

**Comment by Midnite59** [comment]
Does anyone have a estimated time for skaterhax on us consoles?

**Comment by zoogie** [comment]
Very soon, although I might have to just skip hbmenu access and do a ROP install of unSAFE_MODE.
It's a different way to get to the same place: boot9strap. So it shouldn't be a big loss.

**Comment by Midnite59** [comment]
Thanks for replying so quickly. Forgive me for being kind of dumb, but would the exploit just change the wifi access points for unSAFE_MODE skipping the homebrew launcher? Please let me know if I got this right or correct me if I'm wrong.

**Comment by zoogie** [comment]
Yeah that's the gist of it, but uSM install would only be for new3ds 11.17 US specifically, since that's the only one giving me issues getting hbmenu launched. Once you successfully do uSM, you can then do hbmenu through luma3ds.

**Comment by zoogie** [comment]
https://github.com/zoogie/super-skaterhax/releases/tag/v1.1

I was able to fix 11.17 US new3ds ropbin, so hbmenu should be launchable now.

**Comment by CahoonR55** [comment]
I am on a new 3DS XL in the US and  trying to click the 'Go go!' link isn't working, the page just stays blank white.

**Comment by massatt212** [comment]
my page is black and saying 'test" i have a old 3ds

**Comment by RPG304** [comment]
Oi

**Comment by gbouthenot** [comment]
Nothing new, I just tried super-skaterhax on an unmodified 11.17.0-50 (EUR) 3ds, and it was successful. Thank to everyone !

**Comment by eiriksateroy** [comment]
@zoogie for EUR consoles. Is it required to stay on UK region or can you later change to another country after the exploit is done? When doing the exploit I had my N3DS set to another country than UK and it didnt work. When I set it to UK, it worked. I changed it to UK as the troubleshooting guide in 3ds.hacks.guide said so. Will changing back to my original country impact anything?

**Comment by tr1p0p** [comment]
Not working for me after multiple attempts, changing date, time and following all insructions.
